Mu2eUtilities/src/HelixTool.cc

```diff
@@ -99,4 +100,22 @@ namespace mu2e {
     _hitRatio = _nStrawHits/expected_faces;
   }
 
+
+  void   HelixTool::dirOfProp(float& Slope, float& Chi2ndof){
+    ::LsqSums4 fitDtDz;
+    const mu2e::ComboHit* hit;
+    double     timeErrSquared(25.);//ns^2
```

Comment:
Should this number come from Conditions?  Or from the hit itself?  Note that ComboHit provides the error on the TOT time.

Reply:
thanks for noticing it. Could you please point me to the function you are referring to? I only see the `transErr` and `wireErr` functions.

Reply:
Thanks for updating.